libexec/pyenv

```diff
@@ -76,23 +76,27 @@ export PYENV_DIR
 
 shopt -s nullglob
 
-bin_path="$(abs_dirname "$0")"
-for plugin_bin in "${bin_path%/*}"/plugins/*/bin; do
+PYENV_INSTALL_PATH="$(abs_dirname "$0")"
```

Comment:
Maybe `PYENV_PREFIX`?

It's canonically called "prefix" in UNIX (and GNU autotools and CPython's `configure` in particular) because in the FHS, there's no such thing as "install path", as many applications can be installed alongside. But there are possible autonomous subtrees, with the root of such subtree being "prefix".

Perhaps also with a leading underscore to show that this is an internal envvar that is not supposed to be user-overridable (unlike e.g. `PYENV_ROOT`) so we can overwrite it with impunity.